doc/admin-guide/files/records.config.en.rst

```diff
@@ -3424,10 +3424,10 @@ HTTP/2 Configuration
    :reloadable:
    :overridable:
 
-   Specifies how long Traffic Server keeps connections to origin servers open
-   if the transaction stalls. Lowering this timeout can ease pressure on the
-   proxy if misconfigured or misbehaving clients are opening a large number of
-   connections without submitting requests.
+   Specifies how long Traffic Server keeps connections to clients open after
+   initial connection if no activity is received on the connection. Lowering
```

Comment:
Reading this again, it seems a little odd "connections to clients open after initial connection if no activity is received". I think i see what you are saying (this timeout happens after we've accepted the connection), but it reads as if it can only happen right after a connection is established (which is not correct). The timeout can happen anytime during a transactions lifetime.

Reply:
You are correct. The wording was a bit strange on re-reading it. Removing the blurb about initial connections, since that is just confusing the wording more. Your interpretation of the meaning is what was actually meant. It reads much more clearly without the statement about initial connections.